cloudinit/sources/azure/errors.py

```diff
@@ -195,3 +195,17 @@ def __init__(self, exception: Exception) -> None:
 
         self.supporting_data["exception"] = repr(exception)
         self.supporting_data["traceback_base64"] = trace_base64
+
+
+class ReportableErrorProxyAgentNotFound(ReportableError):
+    def __init__(self) -> None:
+        super().__init__("azure-proxy-agent not found")
```

Comment:
Given that no official debian/ubuntu or fedora images yet have a published version of this package, let's provide a bit more context in the error message that tells admin's what triggered this attempt, and 


```suggestion
        super().__init__(
            "Unable to activate ProvisionGuestProxyAgent."
            " azure-proxy-agent package not found"
        )
```

Reply:
done